.travis.yml

```diff
@@ -12,9 +12,12 @@ env:
 
 before_install:
   - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && openssl aes-256-cbc -K $encrypted_26c593b079d9_key -iv $encrypted_26c593b079d9_iv -in .gitcookies.enc -out .gitcookies -d || true'
+  # Download and install dep
+  - curl -sI https://github.com/golang/dep/releases/latest | grep -Fi Location  | tr -d '\r' | sed "s/tag/download/g" | awk -F " " '{ print $2 "/dep-linux-amd64"}' | wget --output-document=$GOPATH/bin/dep -i -
+  - chmod +x $GOPATH/bin/dep
 
 install:
-  - go get -t ./...
+  - make dependencies
```

Comment:
If we're commiting the `vendor` directory, we don't need to run dep in CI at all don't we?

Reply:
Yes and no because we have to validate that the dependencies are in the vendor folder.
I think I need to validate the git state after the `dep ensure`

Reply:
That makes sense!

Instead though, we should use: `dep status -v` which will panic/error on a missing or outdated dependency.